srcpkgs/volta/template

```diff
@@ -0,0 +1,77 @@
+# Template file for 'volta'
+pkgname=volta
+version=0.7.1
+revision=2
+archs="x86_64 i686"
```

Comment:
This should not be limited unless there's an incredibly good and documented reason.

Reply:
I did not have the time to test other architectures. I'm also not sure if those are actually supported by volta (because it needs to fetch binaries for the dependencies it manages, so if there are no binaries for those systems available, it won't work anyways).

Reply:
I just confirmed that other architectures are currently not supported. Official support is only for x86_64, but i686 seems to work, too.